.env.example

```diff
@@ -19,6 +19,7 @@ DEPLOYER_PRIVATE_KEY=
 #ARCHIVE_NODE_opsepolia=https://sepolia.optimism.io
 #ARCHIVE_NODE_opmainnet=https://opt-mainnet.nodereal.io/v1/<YOUR_KEY_HERE>
 ARCHIVE_NODE_basesepolia=https://sepolia.base.org
+ARCHIVE_NODE_basemainnet=https://mainnet.base.org
```

Comment:
Why don't we use a similar node URL, as shown [here](https://github.com/VenusProtocol/protocol-reserve/pull/116/files#diff-a3046da0d15a27e89f2afe639b25748a7ad4d9290af3e7b1b6c1a5533c8f0a8cR24)